tests/AbstractServiceTest.php

```diff
@@ -8,9 +8,7 @@
 
 abstract class AbstractServiceTest extends TestCase
 {
-    const ENTITY_NS = 'BBC\ProgrammesPagesService\Domain\Entity\\';
     const REPOSITORY_NS = 'BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\\';
```

Comment:
you got rid of the other namespaces, can't you do the same for this one?

Reply:
Yes!, is my next PR (is ready)

Reply:
ah great news. this is good to go then.